Makefile

```diff
@@ -191,6 +191,8 @@ install: go.install
 
 check test: go.test
 
+lint: go.lint
```

Comment:
you should probably update the .help target too: https://github.com/rook/rook/blob/master/Makefile#L239